src/NServiceBus.AcceptanceTests/Reliability/Outbox/When_sending_from_a_non_dtc_endpoint.cs

```diff
@@ -6,19 +6,20 @@
     using Configuration.AdvanceExtensibility;
     using EndpointTemplates;
     using NUnit.Framework;
-    using ScenarioDescriptors;
 
     public class When_sending_from_a_non_dtc_endpoint : NServiceBusAcceptanceTest
     {
         [Test]
-        public Task Should_store_them_and_dispatch_them_from_the_outbox()
+        public async Task Should_store_them_and_dispatch_them_from_the_outbox()
         {
-            return Scenario.Define<Context>()
+            Requires.OutboxPersistence();
```

Comment:
hm, maybe something like Requires.OutboxPersistence<EndpointDefinition>(); ? although currently not doable, we could try to expose transport and persistence configuration on the endpoint configuration and access them in the RequireChecks which could be a way to remove the "Capabilities" partial class again.